src/main/java/org/myrobotlab/service/IntegratedMovement.java

```diff
@@ -573,7 +573,7 @@ public void setDHLink(String arm, ServoControl servo, double d, double theta, do
       dhArm.addLink(dhLink);
       engine.setDHRobotArm(dhArm);
       engines.put(arm, engine);
-      servo.subscribe(getName(), "publishAngles", servo.getName(), "onIMAngles");
+      // servo.subscribe(getName(), "publishAngles", servo.getName(), "onIMAngles");

```

Comment:
need a way to publish the solved angles from IK/IM service to the appropriate servo.  it should probably invoke moveTo .. intead of onIMAngles.  (what ever that method was..)

Reply:
absolutely ...
this should have used "command" messages, instead it used status message.  Specifically, status messages that were only generated on speed control from an Arduino. 
It needs to change to command messages.